tensorflow/python/kernel_tests/string_split_op_test.py

```diff
@@ -21,6 +21,7 @@
 import numpy as np
 
 from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import errors
```

Comment:
Do you need to import `errors` or can you use `errors_impl` like the other tests?